scripts/importer_engine_support_check.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generates configs from the full example, inits all modules, and checks if any resources are not supported by the importer.
+// Meant to be run from the scripts/ folder like so:
+// go run importer_engine_support_check.go
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfengine"
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/tfimport"
+)
+
+func main() {
+	full := "../examples/tfengine/full.hcl"
+	if _, err := os.Stat(full); os.IsNotExist(err) {
+		log.Fatalf("example file %v does not exist: %v", full, err)
+	}
+
+	log.Printf("Creating tmpdir for outputting the configs")
+	tmp, err := ioutil.TempDir("", "")
+	if err != nil {
+		log.Fatalf("ioutil.TempDir = %v", err)
+	}
+	defer os.RemoveAll(tmp)
+
+	log.Printf("Generating configs from %v to %v", full, tmp)
+	if err := tfengine.Run(full, tmp); err != nil {
+		log.Fatalf("tfengine.Run(%q, %q) = %v", full, tmp, err)
+	}
+
+	log.Printf("Converting all Terraform backend blocks to local")
+	path := filepath.Join(tmp, "live")
+	tfengine.ConvertToLocalBackend(path)
+
+	log.Printf("Initializing all modules in order to create and fill the .terraform/ dirs.")
+	plan := exec.Command("terragrunt", "plan-all")
```

Comment:
do you need to run plan-all to init all modules. Does init-all work?

Reply:
From what I can tell init-all doesn't work. It fails when it tries to init the root (I think?) and it says it initialized an empty directory.

Reply:
Oh so I guess you are trying to actually use the plan from the console output. So then maybe update the `log.Printf("Initializing all modules in order to create and fill the .terraform/ dirs.")` which makes me think you just need to init the modules and use the contents in `.terraform` folder.

Reply:
You're actually right, that's exactly what I'm doing. I'd like to init and use the contents of the .terraform/ folders (multiple of them). As far as I can tell, `plan-all` does actually do what I would expect `init-all` to do, but it doesn't fail.

Reply:
Ok. Then consider using cmdRun() as you don't really use the output plan, and add a comment why using `plan-all` here.